### ISSUE TITLE
[BUG] items not loading in shops

### ISSUE THREAD

**Issue by ideesans** [pokeclicker/pokeclicker#2510]
**Info**
 - OS: [e.g. Windows 10]
 - Browser [e.g. Chrome, Safari, Firefox]
 - Version [e.g. `v1.0.0`]

**Description:**
 When going in poke marts only pokéballs, greatballs and ultraballs load but all the other items just don't load and i can't buy them. This is the case in every pokémart in every region, including the pokémart in the shortcut menu. Instead, there is ony a square with no image with "item name" and sold out" written

**Steps to Reproduce:**

1. Open a pokémart
2. Items don't load

**Expected Behavior:**
 Normally items should be available and on display

**Screenshots:**

![image](https://user-images.githubusercontent.com/108224378/175816540-ffc47d12-11fc-4764-8939-cc7a6c03294a.png)

**Additional Information:**

**Comment by MacWarrior** [comment]
Hello,

Same issue here ; on Brave 1.40.107.

Here are the JS errors : 
``` 
Unable to process binding "template: function(){return { name:'currencyTemplate',data:{ amount:App.game.wallet.currencies[$data](),currency:$data,reduced:true}} }"
Message: App.game.wallet.currencies[$data] is not a function
    at template (eval at parseBindingsString (knockout-latest.js:74:172), <anonymous>:3:141)
    at init (knockout-latest.js:129:200)
    at knockout-latest.js:79:94
    at Object.G (knockout-latest.js:41:458)
    at knockout-latest.js:79:75
    at Object.D (knockout-latest.js:11:52)
    at p (knockout-latest.js:78:408)
    at k (knockout-latest.js:76:170)
    at m (knockout-latest.js:76:52)
    at k (knockout-latest.js:76:228)
```

```
Unable to process binding "attr: function(){return { disabled:!$data.isAvailable()} }"
Message: Cannot read properties of undefined (reading 'isAvailable')
    at attr (eval at parseBindingsString (knockout-latest.js:74:172), <anonymous>:3:227)
    at update (knockout-latest.js:96:355)
    at a.$.l (knockout-latest.js:79:379)
    at Function.yd (knockout-latest.js:55:359)
    at Function.zd (knockout-latest.js:55:45)
    at Function.ha (knockout-latest.js:54:384)
    at a.o.a.$ (knockout-latest.js:52:147)
    at knockout-latest.js:79:366
    at Object.D (knockout-latest.js:11:52)
    at p (knockout-latest.js:78:408)
```

```
Unable to process binding "template: function(){return { name:'currencyTemplate',data:{ amount:App.game.wallet.currencies[$data](),currency:$data,reduced:true}} }"
Message: App.game.wallet.currencies[$data] is not a function
    at template (eval at parseBindingsString (knockout-latest.js:74:172), <anonymous>:3:141)
    at init (knockout-latest.js:129:200)
    at knockout-latest.js:79:94
    at Object.G (knockout-latest.js:41:458)
    at knockout-latest.js:79:75
    at Object.D (knockout-latest.js:11:52)
    at p (knockout-latest.js:78:408)
    at k (knockout-latest.js:76:170)
    at m (knockout-latest.js:76:52)
    at k (knockout-latest.js:76:228)
```

```
Unable to process binding "attr: function(){return { disabled:!$data.isAvailable()} }"
Message: Cannot read properties of undefined (reading 'isAvailable')
    at attr (eval at parseBindingsString (knockout-latest.js:74:172), <anonymous>:3:227)
    at update (knockout-latest.js:96:355)
    at a.$.l (knockout-latest.js:79:379)
    at Function.yd (knockout-latest.js:55:359)
    at Function.zd (knockout-latest.js:55:45)
    at Function.ha (knockout-latest.js:54:384)
    at a.o.a.$ (knockout-latest.js:52:147)
    at knockout-latest.js:79:366
    at Object.D (knockout-latest.js:11:52)
    at p (knockout-latest.js:78:408)
```

Best regards,
MacWarrior.

**Comment by Croustyzakk** [comment]
I have the same problem : 
all the shops are working normally except the one called "survival area" in Sinnoh. when I click on the pokémart in this area, all the other shops also start bugging, preventing me from accessing things other than balls.

**Comment by Someonealive-QN** [comment]
This bug is known devs are gonna fix it in the next update

**Comment by Ultima1990** [comment]
Fixed by #2511.
Hopefully.
